### DISCUSSION TITLE
Zeek/Suricata Core Selection  -Accidental bypass

### DISCUSSION THREAD

**Discussion by xfaith** [Security-Onion-Solutions/securityonion#8402] [General] [answered]
Under the advance setup for the # of cores for Zeek/Suricata, if you dont "select" the number of cores, it lets you move on without a notification of no core selected.  I only noticed this when I  got the the review stage.  

I had highlighted # of cores for suricata (did not select it though), and hit enter, which brought me to the next step in the install.   I noticed when i checked the review that I had 0 core assigned to suricata.  

Might be good to have some sort of error check here that makes sure the user selects any of the available numbers.

**Comment by dougburks** [comment, marked as answer]
I've created issue https://github.com/Security-Onion-Solutions/securityonion/issues/8434 for this.
